_posts/blog/2015-11-08-gtac.md

```diff
@@ -15,13 +15,87 @@ date: 2015-11-08T15:39:55-04:00
+
+<figure class="third">
+	<a href="/images/GTAC_2015_harvard.jpg"><img src="/images/GTAC_2015_harvard.jpg" alt="image"></a>
+	<a href="/images/gtac_2015_charles.jpg"><img src="/images/gtac_2015_charles.jpg" alt="image"></a>
+	<a href="/images/gtac_2015_MIT.jpg"><img src="/images/gtac_2015_MIT.jpg" alt="image"></a>
+	<figcaption>Some impressions of Boston.</figcaption>
+</figure>
+
+I also had to spend some time at Primark, because Air Berlin managed to forget my luggage in Berlin and I needed some clothes... But this is a different story.
+
+### upday@Google - My favourite talks
+
+<center>
+<picture class="half">
+	<img src="/images/gtac_2015_upday_at_google.png" alt="image">
+	<figcaption>The Google Campus in Cambridge.</figcaption>
+</picture>
+</center>
+
+Let's get down to business. There were several great talks. The scope of topics ranged from frameworks for mobile testing through test processes up to massive test racks for Google Chromecast Sticks and roboters with highspeed cameras. The following selection of talks does not claim to be complete. But this were the sessions that impressed me the most.
+
+### The Uber Challenge of Cross-Application/Cross-Device Testing
+<iframe width="560" height="315" src="https://www.youtube.com/embed/p6gsssppeT0" frameborder="0" allowfullscreen></iframe>
+<a href="https://docs.google.com/presentation/d/1vYXhkvgLKun72Ix91LQDDWZQdcY5VOBqKVvI1Y6riYo/pub">slides</a>
+
+Uber presented a testing-framework that they implemented to allow ui based testing of interconnected (and cross plattform) apps like their driver and passenger app. If you are in the same situation and try to automate a test that needs communication between two apps (messenger, games, etc) give it try!
+
+### Robot Assisted Test Automation
+<iframe width="560" height="315" src="https://www.youtube.com/embed/oQRrk7S9sUE" frameborder="0" allowfullscreen></iframe>
+<a href="https://docs.google.com/presentation/d/1DxEX7d3UZKFv-ekHzt1-eRCWtlR8uAyXt6ra1B14IGQ/pub">slides</a>
+
+Fanzy robot hands and fingers combined with a high speed camera that allow reproducible tests on every touchscreen. There main purpose is to test the accuracy and speed of displays and the devices in general. But there are so cool, I would love to have one of these things at work.
+
+### Chromecast Test Automation
+<iframe width="560" height="315" src="https://www.youtube.com/embed/TfAVJLhb_k0?list=PLSIUOFhnxEiCWGsN9t5A-XOhRbmz54IS1" frameborder="0" allowfullscreen></iframe>
+<a href="https://docs.google.com/presentation/d/1gBGS5UFyyR0mcwg990I1YDOFQwAhL-nmonNPGvGYnlk/pub">slides</a>
+
+Did you ever asked yourself how Google tests hardware like the chromecasts? Well this talk gives you the answer. In a Google way - means in a massive scale with a lot of intelligence behind it.
```

Comment:
Did you ever ask yourself .... like the chromecast?
Well, ...
The Google way - Massive scale with a lot of intelligence.